pkg/agent/informant.go

```diff
@@ -555,6 +556,11 @@ func doInformantRequest[Q any, R any](
 	path string,
 	reqData *Q,
 ) (_ *R, statusCode int, _ error) {
+	result := "<internal error>"
```

Comment:
I think I might just key them on status code (potentially just rounded down to the nearest 100?) rather than these strings, but it's fine